model/utils.py

```diff
@@ -259,23 +254,48 @@ def parse_packets(flp, packet_size_B, direction="data"):
     assert direction in dir_opts, \
         f"\"direction\" must be one of {dir_opts}, but is: {direction}"
 
+    folder = flp[:flp.rindex("/")]
```

Comment:
`path.basename(flp)`.

Reply:
Good point! That's much better than rindex